src/cli/api/apihelper.go

```diff
@@ -379,6 +383,77 @@ func (helper *APIHelper) GetMetrics(metricName string, startTime, endTime int64,
 
 }
 
+func (helper *APIHelper) GetAggregatedMetrics(metricName string, startTime, endTime int64, desc bool, page uint64) (bool, [][]string, error) {
+
+	hasNext := false
+	if page <= 1 {
+		err := helper.CheckHealth()
+		if err != nil {
+			return hasNext, nil, err
+		}
+	}
+
+	baseURL := helper.Endpoint.URL
+	queryMetricURL := strings.Replace(AggregatedMetricPath, "{appId}", helper.Client.AppId, -1)
+	queryMetricURL = strings.Replace(queryMetricURL, "{metric_type}", metricName, -1)
+	requestURL := fmt.Sprintf("%s%s", baseURL, queryMetricURL)
```

Comment:
use path package to handle the path join

Reply:
@boyang9527 
requestURL := path.Join(baseURL, queryMetricURL) will prune the first "/" of the scheme "http://", so result will be: "http:/127.0.0.1:50529/v1/apps/fakeAppId/aggregated_metric_histories/memoryused" and the request fails.

Reply:
or use url package,  my concern is sometimes you passed into baseURL which ends with "/"

Reply:
@boyang9527 fixed in https://github.com/cloudfoundry-incubator/app-autoscaler-cli-plugin/pull/10/commits/2677630841f8ff807bb043bbf64c6a770561840d